R/zchunk_LA1011.en_bal_adj.R

```diff
@@ -1,88 +1,216 @@
-#' @author YourInitials CurrentMonthName 2017
-#' @export
-module_energy_LA1011.en_bal_adj_DISABLED <- function(command, ...) {
+#' @author JDH July 2017
+module_energy_LA1011.en_bal_adj <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "common/iso_GCAM_regID",
              FILE = "energy/calibrated_techs",
              FILE = "energy/EIA_RFO_intlship_kbbld",
              FILE = "energy/EIA_TOT_intlship_kbbld",
-             FILE = "energy/EIA_ctry",
+             FILE = "energy/mappings/EIA_ctry",
              FILE = "energy/A22.globaltech_coef",
              "L101.en_bal_EJ_R_Si_Fi_Yh_full"))
   } else if(command == driver.DECLARE_OUTPUTS) {
     return(c("L1011.en_bal_EJ_R_Si_Fi_Yh",
              "L1011.in_EJ_ctry_intlship_TOT_Yh"))
   } else if(command == driver.MAKE) {
 
+    # silence package check
+    technology <- minicam.energy.input <- sector <-
+      fuel <- supplysector <- subsector <- GCAM_region_ID <- iso <- value.x <-
+      year <- value <- Country <- value_TOT <- value_diff <- value_RFO <- value.y <- NULL
+
+
     all_data <- list(...)[[1]]
 
     # Load required inputs
     iso_GCAM_regID <- get_data(all_data, "common/iso_GCAM_regID")
     calibrated_techs <- get_data(all_data, "energy/calibrated_techs")
     EIA_RFO_intlship_kbbld <- get_data(all_data, "energy/EIA_RFO_intlship_kbbld")
     EIA_TOT_intlship_kbbld <- get_data(all_data, "energy/EIA_TOT_intlship_kbbld")
-    EIA_ctry <- get_data(all_data, "energy/EIA_ctry")
+    EIA_ctry <- get_data(all_data, "energy/mappings/EIA_ctry")
     A22.globaltech_coef <- get_data(all_data, "energy/A22.globaltech_coef")
     L101.en_bal_EJ_R_Si_Fi_Yh_full <- get_data(all_data, "L101.en_bal_EJ_R_Si_Fi_Yh_full")
 
+    EIA_RFO_intlship_kbbld %>%
+      gather(year, value, -Country) %>%
+      mutate(year = as.integer(year)) -> EIA_RFO_intlship_kbbld
+
+    EIA_TOT_intlship_kbbld %>%
+      gather(year, value, -Country) %>%
+      mutate(year = as.integer(year)) -> EIA_TOT_intlship_kbbld
+
     # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # NOTE: there are 'match' calls in this code. You probably want to use left_join_error_no_match
-    # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
-    # NOTE: This code uses vecpaste
-    # This function can be removed; see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
+
+    # MODIFICATIONS TO IEA ENERGY BALANCES
+    # Subset only the relevant years and combine OECD with non-OECD
```

Comment:
Please delete this comment about combining OECD and non-OECD (it was carried over from a different code file in the old data system).